File: test/export/js/09-funciones.js
function sin_titulo() {
	var s, x;
	x = 1.5;
	document.write(Math.sqrt(x),'<BR/>');
	document.write(Math.abs(x),'<BR/>');
	document.write(Math.log(x),'<BR/>');
	document.write(Math.exp(x),'<BR/>');
	document.write(Math.sin(x),'<BR/>');
	document.write(Math.cos(x),'<BR/>');
	document.write(Math.tan(x),'<BR/>');
	document.write(Math.asin(x),'<BR/>');
	document.write(Math.acos(x),'<BR/>');
	document.write(Math.atan(x),'<BR/>');
	document.write(Math.round(x),'<BR/>');
	document.write(Math.trunc(x),'<BR/>');
	document.write(Math.floor(Math.random()*15),'<BR/>');
	s = "Hola";
	document.write(s.length,'<BR/>');
	document.write(String.toLowerCase(s),'<BR/>');
	document.write(String.toUpperCase(s),'<BR/>');
	document.write(String.substring(s,1,3),'<BR/>');
	document.write(String.concat(s," Mundo"),'<BR/>');
	document.write(String.concat("Mundo ",s),'<BR/>');
	document.write(Number("15.5"),'<BR/>');
	document.write(String(15.5),'<BR/>');
}

File: test/export/java/09-funciones.java
import java.io.*;
import java.math.*;
public class sin_titulo {
	public static void main(String args[]) {
		String s;
		double x;
		x = 1.5;
		System.out.println(Math.sqrt(x));
		System.out.println(Math.abs(x));
		System.out.println(Math.log(x));
		System.out.println(Math.exp(x));
		System.out.println(Math.sin(x));
		System.out.println(Math.cos(x));
		System.out.println(Math.tan(x));
		System.out.println(Math.asin(x));
		System.out.println(Math.acos(x));
		System.out.println(Math.atan(x));
		System.out.println(Math.round(x));
		System.out.println(Math.floor(x));
		System.out.println(Math.floor(Math.random()*15));
		s = "Hola";
		System.out.println(s.length());
		System.out.println(s.toLowerCase());
		System.out.println(s.toUpperCase());
		System.out.println(s.substring(1,3));
		System.out.println(s.concat(" Mundo"));
		System.out.println("Mundo ".concat(s));
		System.out.println(String.valueOf("15.5"));
		System.out.println(Double.toString(15.5));
	}
}
